src/main/scala/fr/brouillard/gitbucket/h2/controller/H2BackupController.scala

```diff
@@ -27,7 +42,7 @@ class H2BackupController extends ControllerBase with AdminAuthenticator {
   // private val defaultBackupFile:String = new File(GitBucketHome, "gitbucket-database-backup.zip").toString;
 
   def exportDatabase(exportFile: File): Unit = {
```

Comment:
With the current design (moving functional methods to the companion object), this method also can be moved to the companion object. `Connection` can be taken as an argument.

Reply:
I hadn't made any changes to `exportDatabase` and was trying to minimize the scope of the code changes while still having good hygiene (like writing tests) - so I did the minimum refactor to have tests.

Testing `exportDatabase` is a bit more involved since it would require calling a live database. I didn't want to change that area of the code and thus become responsible for it.

Reply:
Separately, I feel like there's some design or constraint issues with how backup file names are dealt with.

It seems problematic that an external service can pass a `dest` with an absolute (from root `/`) folder or an upwardly-mobile relative folder ( `../../somewhere-else`) to a path on the GitBucket server. I think the path should be, at minimum, constrained to a subfolder that's determined by the GitBucket server configuration.

Reply:
Yeah, that is true. Also, I found that Scalatra actions provided by this plugin don't have permission checking although usage of these actions should be restricted to admin only. Anyway, they are out of the scope of this pull request.